src/utils/api-utils.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export default function FetchAllArticles (){
+    return axios
+    .get("https://newsflash-e6p1.onrender.com/api/articles")
+    .then((response) => {
+      return response.data.articles
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+}
```

Comment:
You can make use of axios.create and baseURL here so you don't need to keep putting in the full URL, take a look at the notes

Reply:
refactored using axios.create